app/views/lines/new.html.erb

```diff
@@ -2,7 +2,7 @@
   <div class="col lines-form">
     <h1 class="border-bottom title"><%= t 'lines.new.welcome_to_daria', fund: FUND %></h1>
 
-    <%= bootstrap_form_tag url: lines_path, class: 'line-select-form' do |f| %>
+    <%= bootstrap_form_with url: lines_path, local: true, class: 'line-select-form' do |f| %>
```

Comment:
Is this `local: true` a change, or is it simply explicit on `with` when it was implied on `tag`?

Reply:
the latter -- `bootstrap_form_tag` was local by default, but `bootstrap_form_with` is remote by default. Not a behavior change, just a difference in what we have to declare.